docs/src/model_docs/vertical/sediment.md

```diff
@@ -13,9 +16,9 @@ structure for the soil loss and lateral structure for the transport in overland
 
 Overview of the different processes for a land cell in wflow\_sediment.
 
-![sediment_inland](../images/soilloss-scheme.png)
+![sediment_inland](../../images/soilloss-scheme.png)
 
-## Soil Erosion
+### Soil Erosion
```

Comment:
The level here is not nice as compared to SBM and HBV when you open the page you only see description and reference but not soil erosion and delivry to the river. Could we change the indent level? And delete Description title then.

Reply:
Headers now have a correct level.